src/core/api/public_api.ts

```diff
@@ -2043,6 +2105,11 @@ class Player extends EventEmitter<IPublicAPIEvent> {
    */
   private _priv_onPlaybackEvent(event : IInitEvent) : void {
     switch (event.type) {
+      case "playback-position":
+        if (this._priv_lastContentPlaybackInfos !== undefined) {
+          this._priv_lastContentPlaybackInfos.lastPlaybackPosition = event.value;
+        }
```

Comment:
The API already listen to clock updates for "positionUpdate" event (see when `_priv_triggerPositionUpdate` is called)